tests/helpers.py

```diff
@@ -460,3 +485,38 @@ def get_cephfs_name():
     )
     result = CFS.get()
     return result['items'][0].get('metadata').get('name')
+
+
+def get_all_pvs():
+    """
+    Gets all pv in given namespace
+    Returns:
+         dict: Dict of all pvc in namespaces
+    """
+    ocp_pv_obj = ocp.OCP(
+        kind=constants.PV, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    output = ocp_pv_obj.get()
+    return output
+
+
+@retry(AssertionError, tries=10, delay=5, backoff=1)
+def validate_pv_delete(sc_name):
```

Comment:
1) validate_pv_deleted
2) this code is very risky - find a way to pass the list of pvs to validate